addons/io/org.openhab.io.imperihome/src/main/java/org/openhab/io/imperihome/internal/ImperiHomeApiServlet.java

```diff
@@ -53,6 +60,11 @@
  *
  * @author Pepijn de Geus - Initial contribution
  */
+@Component(immediate = true, service = HttpServlet.class, configurationPid = "org.openhab.imperihome", property = {
+        org.osgi.framework.Constants.SERVICE_PID + "=org.openhab.imperihome",
```

Comment:
It's now imported. A simple class import because that looks nicer because the auto-formatter would put multiple properties on the same line with a static import. :-)